lib/ngComponents.js

```diff
@@ -0,0 +1 @@
+/*** AUTO-GENERATED BY GULP ***/
```

Comment:
Solved here: 89188e9

Now the module uses .tpl files as internal templates for the generation process:
- lib/ngComponents.tpl is used to generate lib/ngComponents.js
- lib/templates/patterns/template.tpl is used to generate lib/templates/patterns/template.html